ISISNeutronMuon/MDANSE
# Fix MacOS CI/CD

**Description of work**

Related to #147 and #157 

This PR does not fix the issue with VTK, it only fixes up the issues caused by the early merge of #146. Therefore, the build in here should be able to be installed smoothly on all versions of MacOS, after which MDANSE should be able to be started both from Finder and manually, and the command line interface should work without issues as well. However, Molecular Viewer and Elevation, Scalar-Field, and Iso-Surface plotters will crash MDANSE on MacOS 11 and 12. These issues are instead addressed in the Python 3 implementation as shown in #163. However, before that is useable, I think it would be best to release a new patch version with Python 2, i.e. MDANSE 1.5.2.

@sanghamitra-mukhopadhyay and @MBartkowiakSTFC can you please check on your two version of MacOS that this is true?

**To test**

1. Download the MacOS installer from [this run](https://github.com/ISISNeutronMuon/MDANSE/actions/runs/3091893920) and install it.
2. Open the GUI and check that it works (other than the VTK bits)
3. Start the enclosed Python interpreter to check it works.
